packages/expo-image-picker/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### 💡 Others
 
+- Native module on Android is now written in Swift using [Sweet API](https://docs.expo.dev/modules/module-api). ([#17668](https://github.com/expo/expo/pull/17668) by [@bbarthec](https://github.com/bbarthec))
```

Comment:
```suggestion
- Native module on Android is now written in Kotlin using [Sweet API](https://docs.expo.dev/modules/module-api). ([#17668](https://github.com/expo/expo/pull/17668) by [@bbarthec](https://github.com/bbarthec))
```
However, it would be fun to see a Android code written in Swift 😜